### ISSUE TITLE
Seem Django-Raster now support python 3.8 with 0.8.1

### ISSUE THREAD

**Issue by justRishi** [geodesign/django-raster#60]
Can it be that the latest update also added support for python 3.8? Before I was not able to parse raster with 3.8 and had to downgrade to 3.6.  But then again I am also using now latest gdal docker image(osgeo/gdal:ubuntu-small-latest) .

**Comment by yellowcap** [comment]
Yes I think so. The test build in from the github actions uses python 3.8 so it should work. I will add a build matrix at some point to test different python versions.

**Comment by justRishi** [comment]
Thanks, you might want to update then https://pypi.org/project/django-raster/   python version supported.

**Comment by justRishi** [comment]
I accidentally used again previous version of Django-Raster(0.8) with python 3.8 and got Raster algebra errors. When upgrading again to 0.81 the problem was solved again.

I got an error with previous version(0.8)  in combination with python 3.8 when using an url like:
`return `**/raster/tiles**/${this.id}/{z}/{x}/{y}.png?colormap=${this.getFilteredColorMap()}`;`
but not when using  a url like:
``**/raster/algebra**/{z}/{x}/{y}.png?layers=r:0=${this.id},g:1=${this.id},b:2=${this.id}&alpha `;`
